docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
+
+## Prerequisits
+
+### Setup a development environment
+
+1. Install the latest stable versions of [Git](https://git-scm.com/download).
+1. Configure git user credentials for [every respository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-every-repository-on-your-computer) or [single repository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-a-single-repository)
+1. Install the LTS (Long Term Support) version of [Node](https://nodejs.org/en).
+
+### Setup the source repository
+
+Clone the repository, `cd` into the project, install [Lerna](https://github.com/lerna/lerna) and install dependencies:
```

Comment:
I think anyone who is not a collaborator will have to fork and clone the repository. I'm wondering if we should operate under the assumption that this starts when you've already done some of these steps